src/util/display_layout/sizes.cpp

```diff
@@ -298,7 +308,15 @@ void execute_general_stats(config const & cfg)
     if (!layout_file.good() || !layout_file.is_open())
         throw std::logic_error{"Could not open file " + cfg.input.string() + " for reading"};
 
+// https://godbolt.org/z/PeKnxzjn1
+#if defined(__clang__)
+    auto tuple = chopper::layout::read_layout_file(layout_file);
+    auto filenames = std::get<0>(tuple);
```

Comment:
Should move, same in other cpp

Reply:
This moves, looks like a copy? Or is moving it a todo?

Reply:
A todo :)